blib2to3/pgen2/token.pyi

```diff
@@ -64,6 +64,7 @@ if sys.version_info >= (3, 5):
     AWAIT: int
     ASYNC: int
 ERRORTOKEN: int
+COLONEQUAL: int
```

Comment:
Is this perhaps an unrelated change? If so it could be submitted separately.

Reply:
Hrm, that was a mistake. Must have snuck in from my attempt to update the branch to current.